Multisig/Logic/Models/Transaction.swift

```diff
@@ -7,174 +7,38 @@
 //
 
 import Foundation
-import BigInt
-
-// Information about structure described in https://github.com/gnosis/safe/issues/324
-struct Transaction: Decodable, Identifiable, Hashable {
-    var id: Int {
-        return hashValue
-    }
-
-    let safe: String?
-    let to: String?
-    let value: String?
-    let data: String?
-    let operation: Int?
-    let gasToken: String?
-    let safeTxGas: Int?
-    let baseGas: Int?
-    let gasPrice: String?
-    let refundReceiver: String?
-    let nonce: Int?
-    let executionDate: Date?
-    let submissionDate: Date?
-    let modified: Date?
-    let blockNumber: Int?
-    let transactionHash: String?
-    let safeTxHash: String?
-    let executor: String?
-    let isExecuted: Bool?
-    let isSuccessful: Bool?
-    let tokenAddress: String?
-    let ethGasPrice: String?
-    let gasUsed: Int?
-    let fee: String?
-    let origin: String?
-    let dataDecoded: DecodedData?
-    let confirmationsRequired: Int?
-    let confirmations: [TransactionConfirmation]?
-    let signatures: String?
-    let transfers: [Transfer]?
-    let txType: TransactionType?
-
-    var remainingConfirmationsRequired: Int {
-        return max((confirmationsRequired ?? 0) - (confirmations!.count), 0)
-    }
-}
 
+// Transaction domain model based on https://docs.gnosis.io/safe/docs/contracts_tx_execution/#transaction-hash
+// and '/safes/{address}/transactions/' endpoint required data of transaction service
+struct Transaction {
+    // required by a smart contract
+    let to: AddressString?
+    let value: UInt256String
+    let data: DataString?
+    let operation: Operation
+    let safeTxGas: UInt256String
+    let baseGas: UInt256String
+    let gasPrice: UInt256String
+    let gasToken: AddressString?
+    let refundReceiver: AddressString // zero address if no refund receiver is set
+    let nonce: UInt256String
+    // computed based on other properties
+    let safeTxHash: HashString // can be computed from the
```

Comment:
I'd prefer to have comments on a separate line instead of end of line for better editing and readability.

Also, this comment is abruptly ending

Reply:
ok